packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -123,6 +126,79 @@ void _debugRecordDownStream(String channelTypeName, String name,
   _debugLaunchProfilePlatformChannels();
 }
 
+class BackgroundIsolateBinaryMessenger extends BinaryMessenger {
+  final ReceivePort receivePort = ReceivePort();
```

Comment:
can we make this private? I'm a bit wary of having people fiddle with it from outside the class...

Reply:
done